src/sonic-yang-mgmt/yang-models/sonic-vlan.yang

```diff
@@ -38,11 +38,11 @@ module sonic-vlan {
 
 			list VLAN_INTERFACE_LIST {
 
-				key "vlanid ip-prefix";
+				key "vlan_name ip-prefix";
```

Comment:
What is the reason to remove vlanid from the vlan interface list, and replace key from vlanid to vlan_name?